src/main/java/com/shippo/model/Address.java

```diff
@@ -35,8 +35,17 @@ public class Address extends APIResource {
     Object metadata;
     Object messages;
 	ValidationResults validation_results;
+	private boolean test;
 
-    public class ValidationResults {
+    public boolean isTest() {
+		return test;
+	}
```

Comment:
nit, but looks like formatting is off

Reply:
good call, applied standard formatting to that file